app/src/main/java/com/deveddy/clujbike/data/repository/DatabaseHelper.java

```diff
@@ -0,0 +1,21 @@
+package com.deveddy.clujbike.data.repository;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteOpenHelper;
+
+public class DatabaseHelper extends SQLiteOpenHelper {
+
+    public DatabaseHelper(Context context) {
+        super(context, SqlDatabaseContract.DATA_BASE_NAME, null, SqlDatabaseContract.DATA_BASE_VERSION);
```

Comment:
If `database` is one word, shouldn't the constant's name be `DATABASE`? :trollface: